css/site.css

```diff
@@ -1066,4 +1064,48 @@ a.nav-link {
 
 .radio input[type=radio] {
   margin-top: 5px;
+}
+
+.navbar-brand:focus {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
+}
+
+.btn:focus {
+  outline: 5px auto Highlight;
+  outline: 5px auto -webkit-focus-ring-color;
+}
+a.sr-only {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+a.sr-only-focusable:focus,
+a.sr-only:focus {
+  clip: auto;
+  height: 45px;
+  overflow: visible;
+  padding: 8px;
+  position: fixed;
+  top: 100px;
+  white-space: normal;
+  width: 220px;
+  background: #000;
+  color: #fff;
+  outline: 5px auto -webkit-focus-ring-color;
+  z-index: 100;
+}
+
+.tn-prod-list-item__perf-date,
+.tn-prod-list-item__perf-time,
+.tn-prod-list-item__perf-property--datetime,
+.tn-event-listing-view__results-heading,
+.tn-events-list-view__datepicker-divider-label {
+  color: #000000 !important;
```

Comment:
Can you please add a comment explaining why this was necessary? Is it also necessary to use `!important` here (it probably is, but it's best to avoid it if possible)?

Reply:
!important used to take precedence over 3rd party css, and this rule is added to set AA contrast ratio on several elements.